app/assets/stylesheets/admin/application.scss

```diff
@@ -2,3 +2,8 @@
 //= require "../style/css/vendor/fullcalendar.min"
 //= require "../style/css/icons.min"
 //= require "../style/css/app.min"
+//= require "../style/css/style"
+
+.img {
+  background-image: url("assets/style/images/bg_1.jpg")
```

Comment:
Don't use background-image for all class .img on all page (should be for login page)